janitor/functions.py

```diff
@@ -1198,9 +1198,23 @@ def change_type(df, column: str, dtype):
     :param df: A pandas dataframe.
     :param column: A column in the dataframe.
     :param dtype: The datatype to convert to. Should be one of the standard
+    :param ignore_exception: {False, "fillna", "keep_values"}.
         Python types, or a numpy datatype.
     """
-    df[column] = df[column].astype(dtype)
+    if not ignore_exception:
+        df[column] = df[column].astype(dtype)
+    elif ignore_exception == "keep_values":
+        df[column] = df[column].astype(dtype, errors="ignore" )
+    elif ignore_exception == "fillna":
+        # returns None when conversion 
+        def convert(x, dtype):
+            try: 
+                return(dtype(x))
```

Comment:
Minor code style suggestion; `return` statements usually don't need the parentheses.

For convenience, you can commit the change here:

```suggestion
                return dtype(x)
```